Manual.md

```diff
@@ -1302,6 +1303,36 @@ cache during the install/removal of the package
 - `font_dirs`: which should be set to the directory where the package
 installs its fonts
 
+<a id="pkg_remove"></a>
+### Removing a package
+
+Follows a list of things that should be done to help guarantee that a
```

Comment:
Not quite sure if I understand this sentence? Maybe it's supposed to mean `One should follow a list of things to guarantee that...`?

Reply:
A shorter version of 'Here follows'

Reply:
Ah, okay, didn't know one could say it like that (IMO Here follows sounds way better though).